Xamarin.Essentials/Permissions/Permissions.shared.enums.cs

```diff
@@ -23,11 +23,14 @@ enum PermissionType
         Unknown,
         Battery,
         Camera,
+        CalendarRead,
+        CalendarWrite,
```

Comment:
Remove write permissions changes

Reply:
Addressed in:
https://github.com/builttoroam/Essentials/pull/43